_sass/layout/alt-details.scss

```diff
@@ -0,0 +1,46 @@
+// ALT-DETAILS
+//------------------------------------------------
+//------------------------------------------------
+
+.alt-details {
```

Comment:
In general, I think it is better to follow a “utility-first CSS classes” approach, which guarantees that the look and feel is consistent throughout the whole website, and makes the CSS easier to maintain. Here, you introduce new “semantic classes” with specific colors and styles, which are not well integrated with the rest of the website.

You can find more details about “utility-first CSS classes” here: https://adamwathan.me/css-utility-classes-and-separation-of-concerns/

Could you try to make the style of the added content more consistent with the existing content?

Reply:
done